var/spack/repos/builtin/packages/python/package.py

```diff
@@ -470,10 +470,15 @@ def write_easy_install_pth(self, exts):
 
         else:
             with closing(open(main_pth, 'w')) as f:
-                f.write("import sys; sys.__plen = len(sys.path)")
+                f.write("import sys; sys.__plen = len(sys.path)\n")
                 for path in paths:
                     f.write("{0}\n".format(path))
-                f.write("import sys; new = sys.path[sys.__plen:]; del sys.path[sys.__plen:]; p = getattr(sys, '__egginsert', 0); sys.path[p:p] = new; sys.__egginsert = p + len(new)")  # noqa: E501
+                f.write("import sys; new=sys.path[sys.__plen:]; "
+                        "del sys.path[sys.__plen:]; "
+                        "p=getattr(sys,'__egginsert',0); "
+                        "sys.path[p:p]=new; "
+                        "sys.__egginsert = p+len(new)\n")
```

Comment:
It might not be a new line, just an EOL character before the EOF character. Back in the day, I wrote a Python script that output an input file for a Fortran program. The Fortran program couldn't read the file because I had forgotten to add `\n` to the end of the file and it had hit an EOF character before the EOL character.